doc/src/tools/kani.md

```diff
@@ -1,7 +1,124 @@
 # Kani Rust Verifier
 
 The Kani Rust Verifier is a bit-precise model checker for Rust.
-This page will give more details on how to use Kani to verify the standard library.
-You can find more informations about how to install and use Kani in the
-[Kani book](https://model-checking.github.io/kani/).
+Kani is designed to prove safety properties in your code as well as
+the absence of undefined behavior. It uses formal methods under the hood to ensure that
```

Comment:
Add a note saying that is not all undefined behavior. Do we have a documentation with a list of UBs that we can check?

Reply:
it uses "automated reasoning" under the hood... (or model checking)

Reply:
"Some form of UB", does that work?